Domains/2-DisciplinePhysical/StructuralPhysical/StructuralPhysical.ecschema.xml

```diff
@@ -43,7 +43,7 @@
             <Class class="StructuralMemberType"/>
         </Target>
     </ECRelationshipClass>
-    <ECEntityClass typeName="Slab" displayLabel="Slab">
+    <ECEntityClass typeName="Slab" displayLabel="Slab" description="A bis:PhysicalElement; horizontal structural element that is used in building construction">
```

Comment:
Descriptions typically are connected following the layout of the class-hierarchy. E.g. "A horizontal sp:StructuralMember that is used in building construction". Consequently, the description for "StructuralMember" would connect to its base class (i.e. sp:StructuralElement), and so forth.

Reply:
I think it is very helpful to include the horizontal characteristic of slabs to distinguish them from vertical elements such as walls for example.  I also think mentioning something about their geometry to distinguish them from other horizontal elements such as beams would be good too.  For example, a beam has one dimension (length) that is much longer than the other two (section/profile) while a slab has one dimension that is much shorter (thickness) than the other two (boundary/profile).

Reply:
Please check the fixed descriptions